openapi/azure.openai.embeddings/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This is a generated connector from [Azure OpenAI Embeddings API v2023-03-15-preview](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#embeddings/) OpenAPI specification.
+
+The Azure Azure OpenAI Service REST API Embeddings Endpoint will generate one a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
```

Comment:
Azure is repeated twice.

Reply:
Resolved through 140c11110e38029ee8094b18de8f3e70c028c340